### ISSUE TITLE
Build error for debian jessie

### ISSUE THREAD

**Issue by parkmino** [jcorporation/myMPD#289]
myMPD version: 6.4.2

**Describe the bug**

Build error as followings.
I am not sure that it is related to the location of mympd.conf, /usr/etc/mympd.conf not /etc/mympd.conf.
The location of mympd differs only for armel.

```
# ./build.sh pkgdebian
...
[100%] Building C object cli_tools/CMakeFiles/mympd-config.dir/__/src/sds_extras.c.o
Linking C executable mympd-config
make[3]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
[100%] Built target mympd-config
make[2]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
make[1]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
 fakeroot debian/rules binary
cd release; make install DESTDIR=../debian/tmp
make[1]: Entering directory '/root/myMPD-6.4.2/package/build/release'
make[2]: Entering directory '/root/myMPD-6.4.2/package/build/release'
make[3]: Entering directory '/root/myMPD-6.4.2/package/build/release'
make[3]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
[ 97%] Built target mympd
make[3]: Entering directory '/root/myMPD-6.4.2/package/build/release'
make[3]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
[100%] Built target mympd-config
make[2]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
Install the project...
-- Install configuration: "RELEASE"
-- Installing: ../debian/tmp/usr/bin/mympd
-- Installing: ../debian/tmp/usr/etc/mympd.conf
Searching for mpd
	Found /etc/mpd.conf
	Found /usr/bin/mpd
Parsing /etc/mpd.conf
Getting mpd linked libraries /usr/bin/mpd
Writing ../debian/tmp/usr/etc/mympd.conf
-- Installing: ../debian/tmp/lib/systemd/system/mympd.service
-- Installing: ../debian/tmp/usr/bin/mympd-config
make[1]: Leaving directory '/root/myMPD-6.4.2/package/build/release'
mkdir -p debian/tmp/DEBIAN
cp debian/postinst debian/tmp/DEBIAN
cp debian/postrm debian/tmp/DEBIAN
cp debian/conffiles debian/tmp/DEBIAN
dpkg-gencontrol -pmympd
dpkg-gencontrol: warning: File::FcntlLock not available; using flock which is not NFS-safe
dpkg --build debian/tmp ..
dpkg-deb: error: conffile `/etc/mympd.conf' does not appear in package
debian/rules:16: recipe for target 'binary-arch' failed
make: *** [binary-arch] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
ls: cannot access ../mympd_6.4.2-1_*.deb: No such file or directory
Can't find package
Checking package with lintian
warning: the authors of lintian do not recommend running it with root privileges!
```

**Server plattform (please complete the following information):**
 - OS: Debian Jessie
 - OS version: 8
 - Plattform: armel
 - MPD version: 0.20.23

**Comment by jcorporation** [comment]
cmake installs the configuration to /usr/etc/mympd.conf, the debian packaging expect it without the /usr prefix. Please post the compete log, at the beginning of the logs, the calculated install paths from cmake are printed.

**Comment by parkmino** [comment]
Please find the attached file.

[build.log](https://github.com/jcorporation/myMPD/files/4823813/build.log)

**Comment by jcorporation** [comment]
```
ETC_PATH: /usr/etc
BIN_PATH: /usr/bin
VARLIB_PATH: /usr/var/lib/mympd
ASSETS_PATH: /usr/var/lib/mympd/empty
```
Only BIN_PATH is correct. It seems, that cmake constructs the false paths. Is your cmake version older than 3.4?
The handling of special installation prefixes is implemented since 3.4: [changelog](https://cmake.org/cmake/help/v3.4/release/3.4.html)

The CMakeLists.txt of myMPD defines thus a false cmake minimum required version. The 6.5.0 version of myMPD will set the correct minium cmake version.

I do not want handling this case separately, because cmake 3.4 is from 2015 and debian jessy is next month out of support.
